components/image/graphviz/graphviz-php.p5m

```diff
@@ -36,22 +36,18 @@ set name=org.opensolaris.arc-caseid \
     value=PSARC/2011/269
 
 file gv.ini path=etc/php/5.6/conf.d/gv.ini
-file path=usr/lib/graphviz/php/gv.php
-file path=usr/lib/graphviz/php/libgv_php.so
-file path=usr/share/graphviz/demo/modgraph.php
-file usr/share/man/man3/gv.3php path=usr/share/man/man3/gv-php.3
-file path=usr/lib/graphviz/php/gv.so
+
 file path=usr/lib/$(MACH64)/graphviz/php/gv.php
 link path=usr/lib/$(MACH64)/graphviz/php/gv.so target=libgv_php.so
 file path=usr/lib/$(MACH64)/graphviz/php/libgv_php.so
-# These should go in /var/php, but PHP doesn't seem to be able to pick up
-# a second directory configured in extension_dir.
-link path=usr/php/5.6/share/PEAR/gv.php \
```

Comment:
preserve these links, without them I suppose php will not be able to find gv module